### PR TITLE
Fix the error running by CodeQL.

### PR DESCRIPTION
**Pull request by tkoyama010**
### Overview

~~CodeQL is the code analysis engine developed by GitHub to automate security checks. You can analyze your code using CodeQL and display the results as code scanning alerts. For more information about CodeQL, see "[About code scanning with CodeQL](https://docs.github.com/en/code-security/secure-coding/automatically-scanning-your-code-for-vulnerabilities-and-errors/about-code-scanning-with-codeql)." (Quote from [here](https://docs.github.com/en/code-security/code-scanning/automatically-scanning-your-code-for-vulnerabilities-and-errors/about-code-scanning#about-codeql-analysis))~~
Fix the error running by CodeQL.

 

### Details

- None

### CODE REVIEW

**Review by tkoyama010 — COMMENTED** [review]
WFM

**Comment by akaszynski** [comment]
While I think this is a great idea for some projects, I question its usage for pyvista as we're not really web-based and codeql seems to be focused on that domain. This will add another 7 minutes to our build jobs.

Nonetheless, I think we should try it out for a bit. If it's not adding value, we can always remove it later on.

**Comment by tkoyama010** [comment]
That is exactly right. This function is in the GitHub setup and I added it out of curiosity. I welcome to revert it right away if we feel bothered. I love to try everything.

File: README.rst [hunk 1]
```diff
@@ -51,24 +51,25 @@ PyVista
...
+| Activity             | |PyPIact| | |condaact| |            |
++----------------------+-----------+------------+------------+
+| Citation             | |joss|    |  |zenodo|  |            |
++----------------------+-----------+------------+------------+
+| License              | |MIT|     |            |            |
++----------------------+-----------+------------+------------+
+| Community            | |slack|   |  |discuss| |            |
++----------------------+-----------+------------+------------+
+| Formatter            | |black|   |  |isort|   |            |
++----------------------+-----------+------------+------------+
```
Comment (tkoyama010) [inline README.rst:72]:
```suggestion

+----------------------+-----------+------------+
| Deployment           | |pypi|    |   |conda|  |
+----------------------+-----------+------------+
| Build Status         | |GH-CI|                |
+----------------------+-----------+------------+
| Metrics              | |codacy|  |  |codecov| |
+----------------------+-----------+------------+
| Activity             | |PyPIact| | |condaact| |
+----------------------+-----------+------------+
| Citation             | |joss|    |  |zenodo|  |
+----------------------+-----------+------------+
| License              | |MIT|     |            |
+----------------------+-----------+------------+
| Community            | |slack|   |  |discuss| |
+----------------------+-----------+------------+
| Formatter            | |black|   |  |isort|   |
+----------------------+-----------+------------+
```

**Review by tkoyama010 — COMMENTED** [review]
I revert CodeQL running because it is absolutely not needed for this project. (I found only 1 error.)

**Review by akaszynski — APPROVED** [review]
LGTM. Thanks for looking into this @tkoyama010, I've learned something new today!

**Comment by tkoyama010** [comment]
Suggest to merge on 2022/6/14.

**Pull request merged**